src/main/java/com/capitalone/dashboard/settings/ApiSettings.java

```diff
@@ -34,6 +35,8 @@
 
     private String capturePattern;
 
+    private List<String> ignoreEndPoints;
```

Comment:
please define it with a default value i.e. `private List<String> ignoreEndPoints= new ArrayList()`